scripts/ledger.lic

```diff
@@ -250,7 +277,9 @@ module Ledger
     end
 
     def self.main()
-      while (line = get)
+      status_tags
+      while (xml_line = get)
+        line = Ledger::Utility.strip_xml(xml_line.dup)

```

Comment:
Most regex's worked fine without xml, only needed for extracting ID of note handed to you so rather than change all regex to XML version felt easier to make xml and non xml game lines available for match.

Reply:
Uncertain how I feel about using strip_xml vs just updating the regex to appropriate XML version equivalent. Ends up being the same in the end.

Reply:
This way was more me being lazy, at this point it would not take much just to grab the xml versions for the rest. They could likely do with cleanup anyway as they were a bit piecemeal as the list grew over time no doubt and I never validated all the existing ones in game just copied them to a new home and tested via ;send to make sure everything would trigger.